### DISCUSSION TITLE
Widget in Infolist & update from table filters?

### DISCUSSION THREAD

**Discussion by madsem** [filamentphp/filament#12359] [Help]
### Package

Widgets

### Package Version

v3.x

### How can we help you?

Hey gang,

as the title says, is it possible to have a Widget update itself when the table filters change? It's implemented in an InfoList along with a table.

I would be super grateful for any pointers 🤓

PS: This implementation is outside of the Panel Builder.

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/panels/resources/widgets#accessing-page-table-data-in-the-widget
There is an example here: https://demo.filamentphp.com/shop/orders

**Reply by madsem** [reply]
Thanks @leandrocfe That's the panel builder though?

Im using the Infolists standalone with livewire, Tablebuilder etc.

**Reply by leandrocfe** [reply]
Hum, yes. Maybe using livewire events?
https://livewire.laravel.com/docs/events

I did it in V2..
https://filamentphp.com/community/how-to-refresh-widgets-when-table-actions-are-fired#refresh-widgets-automatically

**Reply by madsem** [reply]
Haha, yes I made it work in v2 also, thought there must be a better way now with v3.

Thank you, will look into dispatching Events, maybe that's then the only solution 🤝
